### ISSUE TITLE
Save load doesn't remember "Q nanoframe" and GBC state

### ISSUE THREAD

**Issue by sprunk** [ZeroK-RTS/Zero-K#4942]
Queue "nanoframes only" with Q, save, load. The builder builds full 100%. This is because it's entirely widgetside. GBC also loses state on save load.

**Comment by segatrade** [comment]
> I assume selection rank would be lost too. And apparently the engine loses control groups, which seems retrievable by widgets if the other stuff is.

Yes, control groups CTRL + 1 ... always lost when loaded. This is also bad
